src/Cake.Core/Tooling/ToolSettings.cs

```diff
@@ -39,6 +39,10 @@ public class ToolSettings
         ///     new NuGetSourcesSettings { UserName = "user", Password = "incorrect",
         ///     ArgumentCustomization = args=&gt;args.Append("-StorePasswordInClearText")
         /// });
+        /// // or with multiple arguments:
+        /// MSTest(pathPattern, new MSTestSettings() { ArgumentCustomization = args => { args.Append("/detail:errormessage");
```

Comment:
@robertmuehsig thanks for adding this, I think it makes a lot of sense.  One comment though...

Can you change the `>` to be `&gt;` as per the first example, otherwise **bad things happen** :smile: